assets/app/scripts/services/data.js

```diff
@@ -869,12 +894,19 @@ DataService.prototype.createStream = function(resource, name, context, opts, isR
         }).success(function(data, status, headerFunc, config, statusText) {
           self._listOpComplete(resource, context, data);
         }).error(function(data, status, headers, config) {
-          var msg = "Failed to list " + resource;
-          if (status !== 0) {
-            msg += " (" + status + ")";
+          // mark list op as complete
```

Comment:
Minor thought, wondering if, since we have `_listOpComplete` in `.success()` above, we should move the `.error()` code into another func for symmetry. `_listOpError`, perhaps?